spec/assigner_spec.rb

```diff
@@ -40,6 +40,23 @@
     a.issue_ids.count.should == 1
   end
 
+  it 'should assign comment with ticket when toggled from redmine' do
```

Comment:
Consider wrapping these `it` blocks inside a `context` to get rid of the "when toggled from redmine" part in the description.
http://betterspecs.org/#contexts

Additional benefit: If one day, we decide to drop Redmine, we could just delete this `context` block as it groups tests for a feature we do not need anymore.